homeassistant/util/logging.py

```diff
@@ -121,3 +124,35 @@ def name(self) -> str:
     def name(self, name: str) -> None:
         """Wrap property get_name to handler."""
         self.handler.set_name(name)  # type: ignore
+
+
+def catch_log_exception(func, format_err, *args):
```

Comment:
I don't think that we should pass `args` through it. Since we also create `format_err` , we can already wrap `format_err` in a partial to pass the args.

Reply:
Oh never mind, I did not read the full code. This is fine.